qt_gui/package.xml

```diff
@@ -24,10 +24,11 @@
   <exec_depend>python_qt_binding</exec_depend>
   <exec_depend>python3-catkin-pkg-modules</exec_depend>
   <exec_depend>tango-icon-theme</exec_depend>
+  <exec_depend>ament_index_python</exec_depend>
```

Comment:
Since this is only being used when a developer invokes `dock_widget_title_bar.py` as a script fpr debugging I am inclined to not list it as a dependency in the manifest.

Reply:
Actually after expanding the diff the very same dependency is already defined in line 23.